modules/uavcan_debug/uavcan_debug.c

```diff
@@ -7,7 +7,7 @@
 #include <chprintf.h>
 #include <memstreams.h>
 
-void uavcan_send_debug_msg(uint8_t debug_level, char* source, const char *fmt, ...)
+void uavcan_send_debug_msg(uint8_t debug_level, const char* source, const char *fmt, ...)
```

Comment:
according to our style guide:

const char *fmt

should be:

const char* fmt

Reply:
Technically, we only have a C++ style guide right now. :-)
- I agree with your suggestion tho, and I have changed it.
- I would like for our C++ and C style to be as similar as possible, though there will be some areas where I think they'll have to diverge.
- Hopefully soon we will be able to articulate an Embedded C style guide for the company.